src/javascript/utils/compose.js

```diff
@@ -0,0 +1 @@
+export const compose = (...funcs) => funcs.reduce((a, b) => (...args) => a(b(...args)), arg => arg);
```

Comment:
For future reference we usually use lodash for this, they have a function called flowRight which does the same thing.

Reply:
Thanks I will take a look into that
But being said I thought we are trying to remove the lodash from our repo

Reply:
@vindhya curious what you think would be recommended here? I know we've had some initiatives trying to reduce lodash dependencies on the frontend but this seems to be a valid use case of importing lodash. I believe we can also import specific lodash modules (I believe in this case would be `yarn add lodash.flowRight`) to reduce bundle size.

Reply:
I think it's fine to use the specific `flowRight` package of lodash for this instance as it will be more readable 🙂

Reply:
Updated with lodash appreciated for the suggestion